apps/prairielearn/assets/scripts/question.ts

```diff
@@ -27,7 +27,9 @@ onDocumentReady(() => {
   setupDynamicObjects();
   disableOnSubmit();
 
-  $('.js-submission-body.render-pending').on('show.bs.collapse', loadPendingSubmissionPanel);
+  $<HTMLDivElement>('.js-submission-body.render-pending').on('show.bs.collapse', function () {
+    loadPendingSubmissionPanel.call(this, false).finally(() => {});
```

Comment:
ESLint doesn't like it when we don't handle the Promise that's now being returned from `loadPendingSubmissionPanel`. I need to do something better than this.

I want to switch to using an actual property on the event instead of relying on jQuery's binding of `this`. Maybe I can just rip out jQuery here entirely?

Reply:
Can't rip out jquery yet, since this is a Bootstrap 4 event...

Reply:
I made this change in 99b7353.